### ISSUE TITLE
OwnCloud reports 404 on REPORT

### ISSUE THREAD

**Issue by vacy** [bitfireAT/davdroid#427]
Hello,

I've updated davdroid from 0.6.10 to 0.6.10.1 by Google Play, afterwards the calendar sync doesn't work anymore. It is going to fail reproducable by a http error 404.
Here are the Logs: https://gist.github.com/anonymous/19469ff95a77a5db4b9a 
I am using a OnePlus One with CyanogenMod 11 M11, so it is Android 4.4.4.
I had to anonymize the logs, so the normalized URL is now called: /owncloud/remote.php/caldav/calendars/myusername/1stFirstName-2ndFirstNameSurname/20150103T234653Z-5627_7c15c8b329fe1c3.ics

Do you need something else?
Cheers,
Florian

edit://  CalDAV Server in use: ownCloud 7.0.4 (stable)

**Comment by rfc2822** [comment]
So the server responds with 404 to `REPORT /owncloud/remote.php/caldav/calendars/myusername/1stFirstName-2ndFirstNameSurname/`. Is this URL incorrect? According to the other URLs, it seems to be correct.

**Comment by vacy** [comment]
@rfc2822 thx for reviewing this! :-)
Yes, you're right, here is the http weblog by apache
https://gist.github.com/anonymous/476493b5fb1bf52666d3

**Comment by vacy** [comment]
@rfc2822 edit for the above post, i do not edit because it is a gamechanger of my previos post .... I thought you asked if the URL is correct, but you asked if it is incorrect... however...

I don't think that the URL is incorrect because the previous PROPFIND is working on this URL in the above posted weblog. but it failed on the REPORT method.
How could I investigate further in this?
Owncloud offers this URL by the webinterface as an export:
https://examle.org/owncloud/remote.php/caldav/calendars/myusername/1stFirstName-2ndFirstNameSurname

**Comment by apeiron** [comment]
I am also seeing this. Upgraded to 0.6.10.1 and things stopped working with the error reported above. Other apps like Thunderbird still sync without issue, so I know that it's specific to davdroid.

**Comment by rfc2822** [comment]
I can't reproduce the problem with OwnCloud 7. I guess it's either server misconfiguration that prevents REPORT on this URL or an old OwnCloud version with problems.

**Comment by brortao** [comment]
I'm getting the same issue. Though my log level is set to verbose, REPORT is never mentioned in [my logs](https://gist.github.com/brortao/c7eecbc08bd95b536497). DAVDroid is set to sync all my calendars, but only the default one syncs.

I'm also using ownCloud 7.0.4 and Apache 2.2.22.

**Comment by rfc2822** [comment]
@brortao You have to set the HttpClient and Wire logs to VERBOSE (see https://github.com/bitfireAT/davdroid/wiki/How-to-view-the-logs).

I can't reproduce the problem with OwnCloud 7. I guess it's either server misconfiguration that prevents REPORT on this URL or an old OwnCloud version with problems.

Please report this issue to OwnCloud.

**Comment by vacy** [comment]
Its crazy... It seems that not the update of davdroid caused the issue.
I've a 2nd Phone in sync with another owncloud account, but it is sharing this special calendar across the accounts. It has shown up that the calendar did also not sync 0.6.10 - but it did not throw an error.
I think to remember that the error notifications has been added with 0.6.10.1.
How I fixed that now: exported the calendar in owncloud, deleted calendar, re-added calendar and imported the backup.
Then reconfigured davdroid on the phone -> fixed.
